src/data/javascript-quiz.ts

```diff
@@ -1528,6 +1528,17 @@ const javascriptQuiz = [
       "The toFixed() method converts a number to a string and rounds it to a specified number of decimals.",
     Link: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed",
   },
+  {
+    Question:
+      "In JavaScript, all objects inherit a built-in property from a ___________________.?",
```

Comment:
Since this isn't a question we can remove the `?`
```suggestion
      "In JavaScript, all objects inherit a built-in property from a ___________________.",
```